PFCalEE/include/EventAction.hh

```diff
@@ -53,7 +53,7 @@ private:
   std::vector<SamplingSection> *detector_;
   G4int     evtNb_,printModulo;
 
-  HGCSSGeometryConversion geomConv_;
+  HGCSSGeometryConversion* geomConv_;
```

Comment:
The reason to make it a pointer is to make use of the constructor with a 
nonvoid signature and body.

On 4/5/2016 2:43 AM, amagnan wrote:
>
> In PFCalEE/include/EventAction.hh 
> <https://github.com/pfs/PFCal/pull/17#discussion_r58498508>:
>
> > @@ -53,7 +53,7 @@ private:
> >    std::vector<SamplingSection> *detector_;
> >    G4int     evtNb_,printModulo;
> >
> > -  HGCSSGeometryConversion geomConv_;
> > +  HGCSSGeometryConversion* geomConv_;
>
> is there a reason to make this a pointer ? I personnally prefer the 
> object directly given we normally want only one instance of this 
> class. If we use the pointer we should also add explicitely the 
> "Delete" in the destructor....
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub 
> <https://github.com/pfs/PFCal/pull/17/files/10d68cea58df7a3601955203a93d7cc949b732a4#r58498508>
>